client/my-sites/site-settings/settings-newsletter/EmailsTextSetting.tsx

```diff
@@ -94,6 +71,23 @@ export const EmailsTextSetting = ( { value, disabled, updateFields }: EmailsText
 				<FormSettingExplanation>
 					{ translate( 'The email sent out when someone follows one of your posts.' ) }
 				</FormSettingExplanation>
+
+				<FormLabel htmlFor="confirmation_email_message">
```

Comment:
I think we should hide this whole section for people who never edited the message.

Just not sure yet how...

Reply:
We could return extra fields in the `subscription_options` field from the `/settings` endpoint, changing [this](https://github.com/Automattic/jetpack/blob/main/projects/plugins/jetpack/modules/subscriptions.php#L657-L661) to something like this:

```php
	/**
	 * Return merged `subscription_options` option with module default settings. 
     * Include three extra fields that tell whether any of the fields differ from the default setting.
	 */
	public function get_settings() {
		$default_settings = $this->get_default_settings();
		$current_settings = (array) get_option( 'subscription_options' );

		$result = wp_parse_args( $current_settings, $default_settings );

		// Determine if the current settings differ from the default settings 
		foreach ( $default_settings as $key => $value ) {
			$result[ $key . '_differs_from_default' ] = ( isset( $current_settings[ $key ] ) && $current_settings[ $key ] !== $value );
		}

		return $result;
	}

```

It feels a bit hackish, but it might work 😀

Reply:
Yeah, not sure if worth effort since we're moving to template based (in block editor) anyway at some point soon.

Reply:
Thinking if a simple string comparison in the client side would work...

Reply:
I think we lack the translations there. We could manually bring them in ourselves, but like you say, it's a lot of effort for something temporary.